modules/caddytls/acmeissuer.go

```diff
@@ -92,6 +92,15 @@ func (ACMEIssuer) CaddyModule() caddy.ModuleInfo {
 func (iss *ACMEIssuer) Provision(ctx caddy.Context) error {
 	iss.logger = ctx.Logger(iss)
 
+	// expand email address, if non-empty
+	if iss.Email != "" {
+		email, err := caddy.NewReplacer().ReplaceOrErr(iss.Email, true, true)
```

Comment:
I figure to make `{env.*}` work, right? I can't think of any other placeholder that makes sense here

Reply:
Yep, environment variables is just about all that's useful here.